### DISCUSSION TITLE
Keyball61のキーランドの剥離に関して

### DISCUSSION THREAD

**Discussion by arMaNi843739** [Yowkees/keyball#718] [Q&A] [answered]
Keyball61でトラックボール右側で作成しています
右手側の27番付近のキーランド（赤丸部分）が剥離してしまいました．
どこを接続すれば正常につながるか教えていただきたいです．
![IMG_4716](https://github.com/user-attachments/assets/26e55180-fd9c-4ce1-b832-028b62384eed)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

PCBソケットの生きている右側は通常通り半田付けしてください。その際、このPCBソケットの保持力が弱いので、両面テープ等で張り付けてください。
そして、左側端子について、下記画像の緑点どうしをエナメル線等で接続していただくと復活できると思います。

![KB61keybreakfix](https://github.com/user-attachments/assets/93a464dd-bd30-41c6-a7dd-eb4490b71919)

お手数ですがご対応よろしくお願いいたします。
